index.html

```diff
@@ -52,18 +52,6 @@
 <script src="https://d3js.org/d3.v4.min.js"></script>
 <script src="radar.js"></script>
 
-<!--Azure Application Insight-->
```

Comment:
Heb je bewust de App Insights integratie verwijderd? 
Is dit een integratie van Wigo4it of van iets anders?